ScienceJournal/UI/CropRangeViewController.swift

```diff
@@ -62,11 +60,12 @@ class CropRangeViewController: UIViewController, EditTimestampViewControllerDele
 
   /// Shows an alert that allows the user to manually input a timestamp for the given crop marker.
   ///
-  /// - Parameter marker: A crop marker type.
-  func showTimestampEditAlert(forCropMarkerType markerType: CropOverlayView.MarkerType) {
-    guard let trialCropRange = trialCropRange else {
-      return
-    }
+  /// - Parameters
+  ///   - trialCropRange: The trial's crop range.
+  ///   - markerType: A crop marker type.
+  func showTimestampEditAlert(forTrialCropRange trialCropRange: ChartAxis<Int64>,
+                              andCropMarkerType markerType: CropOverlayView.MarkerType) {
```

Comment:
We could possibly improve this method per our conversation and align it better with the other change you made:
```
showTimestampEditAlert(for: .start, trialCropRange: trialCropRange)
```